update_release_version.sh

```diff
@@ -0,0 +1,7 @@
+sed -ri "s/(TF_VERSION=|tensorflow(-cpu)*(~|=)=|tf-version: \[')[0-9]+[a-zA-Z0-9_.-]+/\1$1/g" \
```

Comment:
Please add copyright info at the top, and relocate to the `tools/` directory to keep project root clean.

Reply:
I moved it to `tools/` but I don't see copyright info in bash scripts. Do you have an example?

Reply:
https://github.com/tensorflow/addons/blob/master/tools/install_deps/buildifier.sh as an example. You're right this is missing from most of our bash scripts... but should still add it I suppose.